settings.gradle

```diff
@@ -76,6 +76,15 @@ List projects = [
   'qa:wildfly'
 ]
 
+File examplePluginsDir = new File(rootProject.projectDir, 'plugins/examples')
```

Comment:
Maybe just make a directory called `example-plugins` instead of keeping it in the plugins directory? I like that the names all line up. Exception for buildSrc, but buildSrc is known special and weird.